### PR TITLE
Various fixes: Validation

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: src/Components/Actions/masking.tsx [hunk 1]
```diff
@@ -5,16 +5,18 @@ import differenceInYears from "date-fns/differenceInYears";
 
 export type MaskType = "PersonalNumber" | "PostalCode";
 
-const PERSONAL_NUMBER_REGEX = /^[0-9]{6}[-]?[0-9]{4}$/;
-const POSTAL_CODE_REGEX = /^[0-9]{3} [0-9]{2}$/;
+const PERSONAL_NUMBER_REGEX = /^[0-9]{6}[0-9]{4}$/;
+const POSTAL_CODE_REGEX = /^[0-9]{3}[0-9]{2}$/;
 
 export const isValid = (m: MaskType | undefined, value: string): boolean => {
+  console.log("mask, value", m, value);
```
Comment (gustafguner) [inline src/Components/Actions/masking.tsx]: Can we remove this?

File: src/Components/Actions/TextAction.tsx [hunk 2]
```diff
@@ -111,8 +117,8 @@ export const TextAction: React.FunctionComponent<Props> = props => {
         >
           <ContinueButton
             onClick={onContinue}
-            disabled={textValue.length === 0 || !isValid(props.mask, textValue)}
-            text={(props.link && props.link.label) || "Nästa"}
+            disabled={!canContinue}
+            text={props.link?.label || "Nästa"}
```
Comment (gustafguner) [inline src/Components/Actions/TextAction.tsx]: 😍TS 3.7

Comment (hedvigoscar) [reply src/Components/Actions/TextAction.tsx]: Nice! I didn't even notice that but I love it

File: src/Components/Actions/masking.tsx (hunk 1 above)
Comment (hedvigoscar) [reply src/Components/Actions/masking.tsx]: yes rofl

**Pull request merged**